public/locales/en-US/send.ftl

```diff
@@ -117,3 +117,146 @@ passwordIsSet = Password set
+legalNoticeSendDataHeader = Data on your device
+legalNoticeSendData = So that you can check status or delete files, basic information about your uploaded files is stored on your local device.  This includes our identifier for the file, the filename, and the file’s download URL. This is cleared if you delete your uploaded file or upon visiting Send after the file expires.
+legalNoticeSendPersonalHeader = Personal data
+legalNoticeSendPersonal = The following is necessary to provide the service
+legalNoticeSendNonPersonalHeader = Non-personal data
+legalNoticeSendNonPersonal = We receive the following to improve our service and performance
+legalNoticeSendThirdPartyHeader = Third Party Services
+legalNoticeSendThirdParty = We use Google Cloud Platform
+legalNoticeSendIPHeader = IP addresses
+legalNoticeSendIP = We receive IP addresses of downloaders and uploaders as part of our standard server logs.  These are retained for 90 days, and for that period, may be connected to activity of a file’s download URL.  Although we develop our services in ways that minimize identification, you should know that it may be possible to correlate the IP address of a Send user to the IP address of other Mozilla services with accounts; and if there is a match, this could identify the account email address.
+legalNoticeSendFXAHeader = Firefox Account
+legalNoticeSendFXA = This is required for authentication only if you wish to upload larger file sizes.  Your Firefox Account record will retain aggregate data on your usage of Send: for example, if you created a Firefox Account in connection with Send, number of files sent and approximate file sizes, and how many times you’ve used the service.
+legalNoticeSendInteractionHeader = Interaction data
+legalNoticeSendInteraction = This includes information such as number of people sending and receiving files, number of files uploaded and approximate file sizes, percentage of file downloaders who become uploaders, how people engage with the website (time spent, clicks, referrer information, site exit path, use of passwords)
+legalNoticeSendTechnicalHeader = Technical data
+legalNoticeSendTechnical = This includes information such as operating system, browser, language preference, country, timestamps, duration for file transfer, reasons for errors, reasons for file expiration.
+# A short representation of a countdown timer containing the number of days, hours, and minutes remaining as digits, example "2d 11h 56m"
+expiresDaysHoursMinutes = { $days }d { $hours }h { $minutes }m
+addFilesButton = Select files to upload
+uploadButton = Upload
+dragAndDropFiles = Drag and drop files
+orClickWithSize = or click to to send up to { $size }
+addPassword = Protect with password
+emailPlaceholder = Enter your email
+signInSizeBump = Sign in to send up to { $size }
+signInButton = Sign in/up
```

Comment:
Why is this using sentence case and the SignOut button title case?

Reply:
good catch